webapp/sass/components/_modal.scss

```diff
@@ -12,7 +12,6 @@
 
     .modal-body {
         max-height: calc(90vh - 62px);
-        overflow: auto;
```

Comment:
Yup it does, the modal body is not what scrolls in that modal. It's something else, and apart from that, and even apart from that, those 3 modals use perfectscrollbar, so they work fine regardless.